assets/less/zenodo-rdm/views/card.overrides

```diff
@@ -0,0 +1,42 @@
+.community-frontpage-cards {
```

Comment:
these directives should be merged with the ones in line 5 and 38, using less notation